site/_data/releases.yaml

```diff
@@ -1,7 +1,7 @@
 stable:
-  name: Darga-3
-  tag: darga-3
-  filename: darga-3
+  name: Darga-4
+  tag: darga-4.1
+  filename: darga-4.1
```

Comment:
How are we handling prerelease versions? e.g. we will be releasing euwe-1-beta1 shortly...It's handled like this on master: https://github.com/ManageIQ/manageiq.org/blob/master/data/deploy_releases.yml (see the commented out section, which we will just uncomment when ready)